### ISSUE TITLE
Do not fail bypass block if there are no blocks to bypass

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#2522]
Noticed this on a page shared on the W3C community slack:
http://af88f81315a0.ngrok.io/recorder.html

Axe-core is flagging this page as requiring a way to bypass repeated blocks of content, which there clearly aren't any of. I don't know how we'd solve for it, but we should figure out some heuristic to turn this into a needs review if we're not sure.

**Comment by dbjorge** [comment]
[One of our samples hit this issue](https://github.com/microsoft/axe-pipelines-samples/pull/419/files#diff-c60bdf5fb0041983d35846bf29fc4bdcaa29af3789edd9b98e5047c665bc2063), too (it used an example page with a very minimal iframe that consisted of just a single non-repeated `<aside>` element, and updating the consumed axe version to 4.0.2 started showing a new `bypass` false positive on `#the-iframe > html`). It's not a problem for us to work around in our sample, but it definitely seems like a false positve issue.

**Comment by WilcoFiers** [comment]
I am leaning towards deprecating this rule. It passes any page with a heading, which is just about every page. Despite providing very little value, it produces false positives.

**Comment by WilcoFiers** [comment]
Talked this through with Dylan, he prefers we have this rule return incomplete instead of a violation. Seems like a reasonable solution, so lets go with that.

**Comment by padmavemulapati** [comment]
Verified with the latest axe-core code base,  seeing `bypass` rule in "incomplete" instead "violation"
test snippet 
```
<div><b>bypass iframe test fail</b></div>
  <a href="foo">foo</a>
 <h1 role="none" id="target">Some stuff and stuff</h1>
```
 ![image](https://user-images.githubusercontent.com/47409579/111956535-e3a65b00-8b10-11eb-88a9-eeaf3023b80b.png)
